### PR TITLE
fix(ie): make IE preserve other browsers' image width styling (#1275)

### PR DESCRIPTION
**Pull request by SpencerWoo**
…e styling

https://github.com/liferay/alloy-editor/issues/1275

Instead of removing all the styles, keep the width (controls sizing) style

### CODE REVIEW

File: src/plugins/dragresize_ie11.js [hunk 1]
```diff
@@ -986,7 +986,11 @@
 
 			const image = wrapper.$.querySelector('img');
 
-			image.removeAttribute('style');
+			let imageStyles = image.getAttribute('style');
+			let widthStyles = /(width:.*?;)/g.exec(imageStyles);
+			let widthStyle = widthStyles[0];
```
Comment (wincent) [inline src/plugins/dragresize_ie11.js]: Seems all of these `let` should be `const`, right?

Also, about that regex: `.*?` is a non-greedy match (good), but `*` implies that zero-width is acceptable and it isn't, is it? ie. `.+?` would be a more accurate specification of what we're looking for. Not that I think we'll get a zero-width value there in practice, just that I think in general it's best to keep regexes as "tight" as possible.

Comment (SpencerWoo) [reply src/plugins/dragresize_ie11.js]: Yeah, they can all be `const`.  Wasn't aware that's the convention but that makes sense.

And right, `.+?` is definitely better here

**Comment by SpencerWoo** [comment]
Didn't know if we were waiting for me to make those changes.

Let me know if there's anything else or when this can be merged so we can handle the LPS.  thanks

**Comment by SpencerWoo** [comment]
@wincent Any update on reviewing this one?  thanks

**Comment by wincent** [comment]
@SpencerWoo: We were looking this over but we had two days out due to office activities here in Madrid. @julien was testing it out and is going to merge it and cut a release today.

**Comment by SpencerWoo** [comment]
awesome thanks guys

**Pull request merged**